pcc/src/lib/addonApiHelper.ts

```diff
@@ -69,6 +70,55 @@ class AddOnApiHelper {
         throw new HTTPNotFound();
     }
   }
+
+  static async createSite(name: string, url: string): Promise<string> {
+    const authDetails = await getLocalAuthDetails();
+    if (!authDetails) throw new UserNotLoggedIn();
+
+    const resp = await axios.post(
+      SITE_ENDPOINT,
+      { name, url, emailList: '' },
```

Comment:
Not sure about what value should be populated for `emailList`.
Need to pass in the body, since it is mandatory field
@kevinstubbs @a11rew

Reply:
An empty string is fine for now.